setup.py

```diff
@@ -634,6 +634,10 @@ def get_extensions():
         mxfp8_src_files_exist = all(os.path.exists(f) for f in mxfp8_sources)
         if mxfp8_src_files_exist and build_for_sm100a:
             print("Building mxfp8_cuda extension")
+            arch_flags = [
```

Comment:
In your compile logs from **before** this change in the PR description, I already see the proper gencodes:

```
...
-DTORCH_EXTENSION_NAME=mxfp8_cuda -gencode=arch=compute_100,code=sm_100 -gencode=arch=compute_100a,code=sm_100a -gencode=arch=compute_120,code=compute_120 -gencode=arch=compute_120,
code=sm_120 -gencode=arch=compute_120a,code=sm_120a -gencode=arch=compute_75,code=sm_75 -gencode=arch=compute_80,code=sm_80 -gencode=arch=compute_86,code=sm_86 -gencode=arch=compute_90,
code=sm_90 -gencode=arch=compute_90a,code=sm_90a 
```